packages/odyssey-design-tokens/src/color/text.json

```diff
@@ -5,6 +5,10 @@
         "comment": "Base text color used for most typography",
         "value": "{color.palette.neutral.900.value}"
       },
+      "heading": {
+        "comment": "Text color for headings",
+        "value": "{color.palette.neutral.900.value}"
+      },
```

Comment:
Let's talk thru the long term vision a bit here to get everyone on the same page - is the idea behind a separate global token (technically L2 in this case) that contains the same value as other text color tokens so that we can carefully map each functional global token to the correct location in component theme reducers? As an example, we'd never map `ColorTextBody` to the `color="heading"` class for `Text` or within other utility classes styling the same type of UI `Heading` etc. This would allow a customer to **globally** theme heading colors one way and other text another way without having to target `[Heading.theme]: { ... }` or `[Text.theme]: { ... }` yes?

I worry a bit about the human judgement required to do these mappings and I also worry about the ambiguity introduced when there are multiple ways to all target this idea, globally using this new property you've setup, and at the L3 level using targeted themes for Heading or Text. Do others have thoughts here?

cc @edburyenegren-okta @andrewberg-okta @arnoldsandoval-okta

Reply:
I think I am agreed. 

Before we make changes on the code side, I think we should have an aligned opinion on which variables we _expect_ to be mapped to each other. In our current case, `color.text.body` and `color.text.heading` share the same value. I'm unclear on whether that is by design (i.e. our desired default for all themes) or if that is a coincidence of our theme's preference.

Our current implementations create some issues here. We often say that L1/Palette values shouldn't need to be touched; however, many components reference them directly. As a customer/developer, it's unclear where and how I would be expected to assemble my theme (or which values will affect what).

Maybe it'd be helpful to have a rule about value cascade. For example: "We expect L2 tokens to map to L1 values. However, L3 tokens should reference L2 values wherever possible."

Reply:
@edburyenegren-okta yes we referenced L1 tokens directly today from L3 tokens. That is problematic for now another place we can improve. As a quick tangent, I've also discussed in the past how it would be cool to introduce another axises for theming where L1 tokens could be themed which would allow folks to tweak palette colors and make updates anywhere those colors are used in L2, all the way down to L3 etc.

Looking at this problem again, the reality is that no one is asking us to theme heading colors separately from body text colors *using only global tokens* today.

We've introduce additional complexity that isn't needed here because we've used functional colors for Text that are limiting (body doesn't encompass headings). Rather than increase the size of our API by introducing `ColorTextHeading` I would simply keep re-using `ColorTextBody` as the values are equivalent and we can work toward a better name in the future.

Reply:
Note that while these values are currently identical, they were different before `0.7.0`, and Visual Design uses a different color for their Headings.

However, if this convo is simply about the addition of a Heading variable, I'm happy to remove it until it's needed.